src/compas/scene/scene.py

```diff
@@ -108,21 +116,32 @@ def add(self, item, parent=None, **kwargs):
         super(Scene, self).add(sceneobject, parent=parent)
         return sceneobject
 
-    def clear(self):
-        # type: () -> None
-        """Clear everything from the current context of the scene."""
+    def clear_context(self, guids=None):
+        # type: (list | None) -> None
+        """Clear all objects from the visualisation context."""
+        clear(guids)
 
-        clear()
+    def clear(self, clear_scene=True, clear_context=True):
+        # type: (bool, bool) -> None
+        """Clear all objects inside the scene.
 
-    def clear_objects(self):
-        # type: () -> None
-        """Clear all objects inside the scene."""
+        Parameters
+        ----------
+        context : bool, optional
+            Clear all objects from the context as well.
```

Comment:
This could indeed have a simple test to cover it, because the `clear`/`clear_context` is already not clear enough by the naming, so, at least adding a test would ensure that this is not broken in the future by accident.